.eslintrc

```diff
@@ -36,6 +36,7 @@
     "import/no-extraneous-dependencies": "off",
     "padded-blocks": "off",
     "import/prefer-default-export": "off",
+    "no-else-return": 0,
```

Comment:
can we make that "only one return statement on any code path" if it exists instead?  I do agree there should be as few wasys as possible to do something, but I also do disagree with this way round (but maybe it's the only option in eslint)